.travis.yml

```diff
@@ -1,19 +1,23 @@
 language: android
 
-# https://github.com/travis-ci/travis-ci/issues/6040#issuecomment-219367943
 android:
   components:
     - tools
     - platform-tools
-    - tools
```

Comment:
Do you think is safe to remove the duplicated "tools", today it works because TravisCI is up-to-date but I won't remove it.